data-model/src/main/scala/za/co/absa/enceladus/model/versionedModel/VersionedSummary.scala

```diff
@@ -16,3 +16,5 @@
 package za.co.absa.enceladus.model.versionedModel
 
 case class VersionedSummary(_id: String, latestVersion: Int)
```

Comment:
Since we are moving this towards ENC v 3 can we have a change that this is not `_id` but `name`?

Also, I am thinking how helpful a description would be at this point.

Reply:
I have created another case class that carries out the change (`_id`-> `name`). But only for API v3, because for V2 the original implementation. had to stay. And actually, the original case class is designed specifically as the mongo response would look like, so it is still useful

Reply:
I think I might've posted the comment under the wrong line of code but what I am getting is 

![image](https://user-images.githubusercontent.com/17467768/166906550-4c9961db-61a2-4de2-93bf-e1fcc366930e.png)

And I expect the `_id` to be `name`. Am I doing something wrongly?

Reply:
Have it.